zerver/openapi/zulip.yaml

```diff
@@ -1310,32 +1169,24 @@ paths:
                         }
                       }
                     }
-  /user_uploads:
-    post:
-      operationId: upload_file
+  /messages/{message_id}:
+    # TODO: move this endpoint back to its original place after
+    # the bug due to which openapi-core matches endpoint greedily
+    # is removed.
```

Comment:
Is that a bug?  That's how URL resolvers usually work; I guess what we want to do is encode that `{message_id}` has to be an integer for the purpose of that matching.

Reply:
Its actually a [closed issue](https://github.com/p1c2u/openapi-core/issues/226) in openapi-core but it is yet to make it to the official release. It matches the endpoints with path parameters in a buggy manner.

Reply:
OK; in that case we can just install a commit of openapi-core.  See https://zulip.readthedocs.io/en/latest/subsystems/dependencies.html

Reply:
I tried it but it turns out the issue is still not fully fixed and fails on `users/{user_id}` and `users/me`. So I will try to reopen or create another issue in the repi.

Reply:
OK, yeah, that's definitely worth at least opening an issue for.